openedx/features/partners/tests/factories.py

```diff
@@ -0,0 +1,23 @@
+from factory.django import DjangoModelFactory
+
+from openedx.features.partners.models import Partner, PartnerUser
+from lms.djangoapps.onboarding.models import Organization
+
+
+class PartnerFactory(DjangoModelFactory):
+    """ Partner factory."""
+    class Meta:
+        model = Partner
```

Comment:
Let's use Faker to populate Model data, It will help creating model objects without adding irrelevant unnecessary data, all the time. For reference you can see factory in this [PR](https://github.com/philanthropy-u/edx-platform/pull/1292)

Reply:
actually here we don't need random data.For partner we have to have a valid label and slug because based on that views are imported in the backend.